### PR TITLE
chore: Move platform relevant instructions to platform folder

### PR DESCRIPTION
**Pull request by ssd71**
### Description

This PR moves the platform relevant instructions to inside the platform folder.
Relevant issue: #37 

##### Checklist

- [x] `npm test` passes
- [x] `npm run build` works
- [x] tests are included
- [x] documentation is changed or added

### CODE REVIEW

**Comment by ssd71** [comment]
Hey @machi1990! I think it's inside the platform folder, I will fix the link to it

**Comment by machi1990** [comment]
Cool.

File: platform/README.adoc [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+1. Go to scripts ./mqtt
+2. Execute docker-compose up
+3. Set MQTT_HOST environment variable in .env file
+
+MQTT_HOST=127.0.0.1
+
+=== Running On OpenShift
+
+Please check link:./openshift[.openshift] folder for more information.
```
Comment (machi1990) [inline platform/README.adoc:194]: We should update the link to the

**Comment by wtrocki** [comment]
Nice. Now we can actualwy remove some of the docs and make docs how to run platform and separate development. Going to do that in separate PR

**Pull request merged**